File: Java/baseballteam/src/Dto/Batter.java
package Dto;

public class Batter {
	private double batCount;
	private double hit;
	private double batavg;
	
	
	public Batter(double batCount, double hit, double batavg) {
		super();
		this.batCount = batCount;
		this.hit = hit;
		this.batavg = batavg;
	}


	public double getBatCount() {
		return batCount;
	}


	public void setBatCount(double batCount) {
		this.batCount = batCount;
	}


	public double getHit() {
		return hit;
	}


	public void setHit(double hit) {
		this.hit = hit;
	}


	public double getBatavg() {
		return batavg;
	}


	public void setBatavg(double batavg) {
		this.batavg = batavg;
	}


	@Override
	public String toString() {
		return "Batter [batCount=" + batCount + ", hit=" + hit + ", batavg=" + batavg + "]";
	}
	
	
	
}

File: kotlin/test2/src/main/kotlin/dto/Batter.kt
package dto

class Batter :Human {
    var batCount:Int = 0
    var hit:Int = 0
    var btaAvg:Double = 0.0

    constructor() : super(0,"",0,0.0)

    constructor(number: Int, name: String, age: Int, height: Double) : super(number, name, age, height){
        this.batCount = batCount
        this.hit = hit
        this.btaAvg = btaAvg
    }

    override fun toString(): String {
        return "$batCount-$hit-$btaAvg"
    }

}